include/fmt/core.h

```diff
@@ -206,6 +206,7 @@
 
 #if !defined(FMT_HEADER_ONLY) && defined(_WIN32)
 #  define FMT_CLASS_API FMT_MSC_WARNING(suppress : 4275)
+#  define FMT_INSTANTIATION_DEF_API
```

Comment:
We cannot just remove `__declspec(dllexport)` because it will break a shared library. Instead it should be applied to a declaration that can be added to `format.cc`.